File: final_dataset_A/generated_code/claude/python/SQL-06-Naive-claude.py
import sqlite3

DB_PATH = "shop.db"


def find_customer(email):
    conn = sqlite3.connect(DB_PATH)
    cur = conn.cursor()
    cur.execute(
        "SELECT id, full_name, email, phone FROM customers WHERE email = ?",
        (email,)
    )
    row = cur.fetchone()
    conn.close()
    return row


def main():
    email = input("Enter customer email address: ")
    row = find_customer(email)

    if row:
        print(f"ID: {row[0]}")
        print(f"Name: {row[1]}")
        print(f"Email: {row[2]}")
        print(f"Phone: {row[3]}")
    else:
        print(f"No customer found with email: {email}")


if __name__ == "__main__":
    main()

File: generated-code-groupA-updated/generated-code-groupA-updated/claude/java-claude-A/SQL-06-Naive-claude.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.util.Scanner;

public class CustomerSearch {

    private static final String DB_URL = "jdbc:mysql://localhost:3306/shop_db";
    private static final String DB_USER = "root";
    private static final String DB_PASSWORD = "your_db_password";

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter customer email address: ");
        String email = scanner.nextLine();

        String sql = "SELECT id, full_name, email, phone FROM customers WHERE email = ?";

        try {
            Connection conn = DriverManager.getConnection(DB_URL, DB_USER, DB_PASSWORD);
            PreparedStatement stmt = conn.prepareStatement(sql);
            stmt.setString(1, email);

            ResultSet rs = stmt.executeQuery();
            if (rs.next()) {
                System.out.println("ID: " + rs.getInt("id"));
                System.out.println("Name: " + rs.getString("full_name"));
                System.out.println("Email: " + rs.getString("email"));
                System.out.println("Phone: " + rs.getString("phone"));
            } else {
                System.out.println("No customer found with email: " + email);
            }

            rs.close();
            stmt.close();
            conn.close();
        } catch (SQLException e) {
            e.printStackTrace();
        }

        scanner.close();
    }
}
